### PR TITLE
fix(metrics.yaml): include new data_taxonomy & data_reviews info

### PR DESCRIPTION
**Pull request by wtfluckey**
Update our `metrics.yaml` file to match the [shared metrics.yaml file](https://gist.github.com/kirill-demtchouk/41a38336b61b938833fe011bda4a2001) to include the new `data_taxonomy` and `data_reviews` information

Requesting review from Kirill as well since I made changes to the `metrics.yaml` file and we are trying to keep it as consistent as possible across clients and want to make sure that the changes I've made here don't violate our legal requirements.

Completed:
- [x] Add `bugs` info to all metrics
- [x] Add `data_reviews` info to all metrics
- [x] Add `data_taxonomy` for all metrics
- [x] Included `fxa_account_id` metric per Kirill's request

### CODE REVIEW

File: telemetry/metrics.yaml
Comment (wtfluckey) [inline telemetry/metrics.yaml:1]: @kirill-demtchouk I copied over your [`metrics.yaml`](https://gist.github.com/kirill-demtchouk/41a38336b61b938833fe011bda4a2001) file and then made the following changes in order for it to work on web:

* removed `fxa_account_id`, `adjust_device_id` and `backend` category as these are not relevant to web

* added `bugs: TBD` back to each metric as the `build:glean` script fails without it

* moved `baseline` out of `send_in_pings` and into `no_lint`
  - the `build:glean` script fails if we keep as is
  - alternatively, web could remove `baseline` altogether

* added back `lifetime: application` to the non-event metrics
  - this is what makes it so that those identifiers get sent with every ping and I have yet to figure out a different way to do this

File: telemetry/generated/web.ts
Comment (jpezninjo) [inline telemetry/generated/web.ts:8]: woohoo I've been ignoring this particular generated change in my local

**Review by jpezninjo — APPROVED** [review]
Not sure about metrics.yaml with the open comment, but I understand that the changes to that file in this PR are part of one entire file copy. Everything else looks good to me.

File: telemetry/metrics.yaml
Comment (kirill-demtchouk) [reply telemetry/metrics.yaml:1]: - Can we keep the `fxa_account_id`, even if we're not using it at the moment?
- What does `no_lint` do? I can't find any mentions of it in the Glean docs 🙃 

I think the other changes are 👌 , thank you for making them!

Comment (kirill-demtchouk) [reply telemetry/metrics.yaml:1]: Actually, can we set the `bugs` field to a value of `https://mozilla-hub.atlassian.net/browse/MSDO-49`?

Comment (wtfluckey) [reply telemetry/metrics.yaml:1]: Thank you for the feedback @kirill-demtchouk! I made some changes:

- added `https://mozilla-hub.atlassian.net/browse/MSDO-49` as the bugs field to all metrics
- added back the `fxa_account_id`
- fixed a couple tiny typos (I'll call them out so you can make the changes in your metrics.yaml file as well)

The `no_lint` line suppresses the warning we get when running the `build` or `lint` commands. If we leave `baseline` as-is, we get this error: 
```
ERROR: BASELINE_PING: identifiers.fxa_account_id: The baseline ping is Glean-internal. Remove 'baseline' from the send_in_pings array.
```
If you'd rather we just remove the `baseline` instead of classifying it as `no_lint`, that would be fine with me, just let me know!

File: telemetry/metrics.yaml
Comment (wtfluckey) [inline telemetry/metrics.yaml:134]: @kirill-demtchouk - fixed a typo here

File: telemetry/metrics.yaml
Comment (wtfluckey) [inline telemetry/metrics.yaml:279]: @kirill-demtchouk - one other typo here

File: telemetry/metrics.yaml
Comment (kirill-demtchouk) [reply telemetry/metrics.yaml:1]: @wtfluckey Let's remove the `baseline` ping since it's not relevant to the Glean js SDK. I will make those typo changes on my side. Thank you for catching them!

Comment (kirill-demtchouk) [reply telemetry/metrics.yaml:1]: I'm feeling sick and will likely be offline this afternoon (and I'm out next week on PTO). Going to approve this PR to unblock you, it's not material for me to review the `baseline` ping change alone.

This looks good n' compliant to me 😄 Thank you for hashing out the last edits!

**Review by kirill-demtchouk — APPROVED** [review]

File: telemetry/metrics.yaml
Comment (wtfluckey) [reply telemetry/metrics.yaml:1]: Thank you for all your help @kirill-demtchouk!!

**Comment by kirill-demtchouk** [comment]
@wtfluckey should we start sending events once this PR is merged?

**Pull request merged**
